public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx

```diff
@@ -85,13 +91,14 @@ export const SingleTopBar = memo(function SingleTopBar({
   );
 });
 
-const getStyles = (theme: GrafanaTheme2) => ({
+const getStyles = (theme: GrafanaTheme2, menuDockedAndOpen: boolean) => ({
   layout: css({
     height: TOP_BAR_LEVEL_HEIGHT,
     display: 'flex',
     gap: theme.spacing(1),
     alignItems: 'center',
-    padding: theme.spacing(0, 1),
+    padding: theme.spacing(0, 1, 0, 0.5),
+    paddingLeft: menuDockedAndOpen ? theme.spacing(3.5) : theme.spacing(0.5),
```

Comment:
We can just leave `padding` as it was before because `paddingLeft` will override that `0.5` right?

Reply:
yeah i guess that's true 👍 i'll put it back